util/codec/bytes.go

```diff
@@ -67,6 +67,14 @@ func EncodeBytes(b []byte, data []byte) []byte {
 	return result
 }
 
+// EncodedBytesLength returns the length of data after encoded
+func EncodedBytesLength(data []byte) int {
```

Comment:
Use `dataLen` as input parameters are better to use?

Reply:
Sure, change commited :)

Reply:
Is there something wrong with jenkins? After the last commit, test fails.